SPEC.md

```diff
@@ -18,31 +18,31 @@ The extension for playlist files is `.blist`
 ### Beatmap
 | Key | Type | Extra Info |
 | - | - | - |
-| `type` | `string enum` | See [Beatmap Types](#beatmap-types) for valid enum values |
+| `type` | `int enum` | See [Beatmap Types](#beatmap-types) for valid enum values |
 | `dateAdded` | `date` | Date when this map was added to the playlist |
-| `key` | `string` | Only present when `type == "key"` |
-| `hash` | `string` | Only present when `type == "hash"` |
+| `key` | `int` | Only present when `type == "key"` |
+| `hash` | `binary` | Only present when `type == "hash"` |
```

Comment:
I would prefer `byte[]` as hash type, I think it's clearer

Reply:
Maybe `byte[20]`?

Reply:
Works for me 👍